services/system/AuthEcSys/include/services/system/AuthEcSys.hpp

```diff
@@ -7,38 +7,71 @@
 
 namespace SystemService
 {
-   struct AuthRecord
-   {
-      psibase::AccountNumber account;
-      psibase::PublicKey     pubkey;
+   namespace AuthEc
+   {  // Using a namespace here disambiguates this class from the one in AuthSys so docs don't get confused
+      /// A record containing the authorization claims needed for an account using this auth service.
```

Comment:
I would probably call it `AuthEcRecord`.

Reply:
Done

Reply:
I actually meant renaming the struct and dropping the namespace.

Reply:
Done